ui/raidboss/data/03-hw/dungeon/xelphatol.txt

```diff
@@ -0,0 +1,139 @@
+### XELPHATOL
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 00:0839:.*is no longer sealed/ window 5000 jump 0
+
+#~~~~~~~~~~~~~~#
+# NUZAL HUELOC #
+#~~~~~~~~~~~~~~#
+
+# -ii 19CB 19CC
+# -ic "Ixali Stitcher" "Airstone"
+
+0 "Start" sync / 00:0839:The Cage will be sealed off/ window 0,1
+7.0 "Short Burst" sync /:Nuzal Hueloc:19C6:/ window 7,5
+15.1 "Wind Blast" sync /:Nuzal Hueloc:19C7:/
+19.2 "Short Burst" sync /:Nuzal Hueloc:19C6:/
+24.3 "Lift" sync /:Nuzal Hueloc:19C9:/ window 24.3,5
+24.8 "--untargetable--"
+
+28.5 "Air Raid" sync /:Nuzal Hueloc:19CA:/ window 5,6
+36.6 "Air Raid" sync /:Nuzal Hueloc:19CA:/
+48.1 "Air Raid" sync /:Nuzal Hueloc:19CA:/
+59.6 "Air Raid" sync /:Nuzal Hueloc:19CA:/ jump 36.6
+71.1 "Air Raid" 
+82.6 "Air Raid"
+94.1 "Air Raid"
+
+100.0 "--sync--" sync / 1E:........:Nuzal Hueloc loses the effect of Hover/ window 100,5
```

Comment:
This is maybe a weird nitpick, but this would be the first 1E line we've synced against in any timeline.  What do you think about using a 22 line instead? Or is there no 22 line here?

Part of my feeling here is that `test_timeline.py` only supports certain line types, and fflogs only has certain line types, and also in the future I have some hope of using network log lines for timelines (implicitly, not in the timeline file itself) and so having fewer log types to handle is a sort of nice goal.

Reply:
There is none. We could in theory wait for an auto-attack.... somewhere? I'm fine with trying to find something else to sync against, but this is the earliest we get to know the floating phase is over.

```
22|2020-05-03T01:12:36.4080000-04:00|400277E4|Floating Turret|19CC|Hot Blast|[PLAYER 1]|[PLAYER 1 ID]|
00|2020-05-03T01:12:36.0000000-04:00|282b||The floating turret uses Hot Blast.|a76b0cd08d41e832b6ac569ec5cfa33a
00|2020-05-03T01:12:36.0000000-04:00|2929||   [PLAYER 1 ID]Balmung takes 633 damage.|ef8c603405355edb168963f92676f5db
37|2020-05-03T01:12:36.9870000-04:00|[PLAYER 1]|[PLAYER 1 ID]|0000C140|18742|30385|10000|10000|0|
00|2020-05-03T01:12:36.0000000-04:00|2ab0||Nuzal Hueloc loses the effect of Invincibility.|8bc410764d917a0e48548b90941fd59f
00|2020-05-03T01:12:36.0000000-04:00|2ab0||Nuzal Hueloc loses the effect of Hover.|
00|2020-05-03T01:12:36.0000000-04:00|2ab0||Nuzal Hueloc loses the effect of Damage Up.|
34|2020-05-03T01:12:37.3880000-04:00|4002779D|Nuzal Hueloc|4002779D|Nuzal Hueloc|01|
38|2020-05-03T01:12:37.3880000-04:00|4002779D|Nuzal Hueloc|003C3C00|167913|320000|8840|8840|0|0|
30|2020-05-03T01:12:37.3880000-04:00|19c|Hover|0.00|4002779D|Nuzal Hueloc|4002779D|Nuzal Hueloc|
30|2020-05-03T01:12:37.3880000-04:00|307|Invincibility|0.00|E0000000||4002779D|Nuzal Hueloc|
30|2020-05-03T01:12:37.3880000-04:00|1bb|Damage Up|0.00|E0000000||4002779D|Nuzal Hueloc|
39|2020-05-03T01:12:38.4140000-04:00|[PLAYER 1]|[PLAYER 1 ID]|
04|2020-05-03T01:12:39.0380000-04:00|400277e4|Floating Turret|0|3c|0|0|
21|2020-05-03T01:12:40.5580000-04:00|4002779D|Nuzal Hueloc|368|Attack|[PLAYER 1 ID]|[PLAYER 1]
```

(Lines cut short for space reasons.)

We could use the "targetable" line instead, if you would prefer, but otherwise we're waiting for something to happen.

Reply:
Yeah, I think the 22 line is preferable to the 1E line.

Reply:
Thanks for noting that, I'm now not even sure why I used `losesEffect` in the first place, `22` is exactly the same timing.